src/test/java/com/cloudbees/jenkins/support/impl/CustomLogsTest.java

```diff
@@ -93,30 +95,31 @@ public void testCustomLogRotation() throws IOException {
         customLogsDir.mkdirs();
 
         // Create dummy log for test1 log recorder
-        try (FileWriter writer = new FileWriter(new File(customLogsDir, "test1.log"))) {
-            writer.write("test1 one");
-        }
-        try (FileWriter writer = new FileWriter(new File(customLogsDir, "test1.log.1"))) {
-            writer.write("test1 two");
-        }
-        try (FileWriter writer = new FileWriter(new File(customLogsDir, "test1.log.2"))) {
-            writer.write("test1 three");
-        }
-
-        // Create dummy log for test2 log recorder
-        try (FileWriter writer = new FileWriter(new File(customLogsDir, "second.test2.log"))) {
-            writer.write("second.test2 one");
-        }
-        try (FileWriter writer = new FileWriter(new File(customLogsDir, "second.test2.log.1"))) {
-            writer.write("second.test2 two");
-        }
-        try (FileWriter writer = new FileWriter(new File(customLogsDir, "second.test2.log.2"))) {
-            writer.write("second.test2 three");
-        }
-
-        try (FileWriter writer = new FileWriter(new File(customLogsDir, "nonRotatedCustomLog.log"))) {
-            writer.write("nonRotatedCustomLog one");
-        }
+        Files.write(Paths.get(customLogsDir.getPath(), "test1.log"), "test1 one".getBytes(), StandardOpenOption.CREATE);
```

Comment:
just
```suggestion
        Files.write(Paths.get(customLogsDir.getPath(), "test1.log"), "test1 one".getBytes());
```
suffices (ditto elsewhere)

Reply:
removed all usages of `StandardOpenOption.CREATE `